lib/renaissance/auctions/auction.ex

```diff
@@ -18,8 +19,8 @@ defmodule Renaissance.Auction do
   @doc false
   def changeset(user, attrs \\ %{}) do
     user
-    |> cast(attrs, @required_fields ++ @optional_fields)
-    |> validate_required(@required_fields)
+    |> cast(attrs, [:title, :description, :user_id, :price, :end_date])
```

Comment:
Why this change? Using the `@required_fields` and `@optional_fields` is the preferred way here. Cuts down on duplication and makes it easier to add new fields.

Reply:
This was an issue with rebasing from the other branch, this will be fixed.